plutus-playground/deployment/nixops/default.nix

```diff
@@ -1,13 +1,16 @@
 let
-  playground = import ../../../. {};
+  plutus = import ../../../. {};
   playgroundMachine = import ./playground.nix;
-  machines = (playground.pkgs.lib.importJSON ./machines.json);
+  meadowMachine = import ./meadow.nix;
+  machines = (plutus.pkgs.lib.importJSON ./machines.json);
   overlays = import ./overlays.nix;
-  playgroundConfig = playground.pkgs.copyPathToStore ./playground.yaml;
+  playgroundConfig = plutus.pkgs.copyPathToStore ./playground.yaml;
   stdOverlays = [ overlays.journalbeat ];
-  options = { inherit stdOverlays machines defaultMachine playground playgroundConfig; };
+  options = { inherit stdOverlays machines defaultMachine plutus playgroundConfig; };
   defaultMachine = (import ./default-machine.nix) options;
   playgroundA = playgroundMachine.mkInstance options machines.playgroundA;
   playgroundB = playgroundMachine.mkInstance options machines.playgroundB;
+  meadowA = meadowMachine.mkInstance options machines.meadowA;
+  meadowB = meadowMachine.mkInstance options machines.meadowB;
 in
-  { inherit playgroundA playgroundB; }
+  { inherit playgroundA playgroundB meadowA meadowB; }
```

Comment:
Seems like this maybe shouldn't be in `playground` any more. Maybe move `deployment` to the top-level?